From ba35b545f464bc03bcce8f6a5d2c74f04872aee8 Mon Sep 17 00:00:00 2001
From: Birger Schacht <birger@rantanplan.org>
Date: Wed, 22 Jan 2020 19:54:03 +0100
Subject: [PATCH] Fix version in buildfile

---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index be081e1..3695f79 100644
--- a/meson.build
+++ b/meson.build
@@ -1,7 +1,7 @@
 project(
 	'swaylock',
 	'c',
-	version: '1.4',
+	version: '1.5',
 	license: 'MIT',
 	meson_version: '>=0.48.0',
 	default_options: [
-- 
2.25.0

