From 98a6fdf7cc7609fb58df7222f6ed3499e2240178 Mon Sep 17 00:00:00 2001
From: Birger Schacht <birger@rantanplan.org>
Date: Fri, 3 May 2019 22:07:41 +0200
Subject: [PATCH] Fix the version in the buildfile

---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 5483d9a..3110e8e 100644
--- a/meson.build
+++ b/meson.build
@@ -1,7 +1,7 @@
 project(
 	'swaylock',
 	'c',
-	version: '1.3',
+	version: '1.4',
 	license: 'MIT',
 	meson_version: '>=0.48.0',
 	default_options: [
-- 
2.20.1

